odoo/addons/base/tests/test_configmanager.py

```diff
@@ -610,15 +607,3 @@ def test_11_auto_stop_after_init_after_test(self):
             with self.subTest(args=args):
                 _, options = self.parse_reset(args)
                 self.assertEqual(options['stop_after_init'], stop_after_init)
-
-    def test_12_without_demo_equal_1(self):
```

Comment:
Note: a related PR is doing this already #188826

Reply:
> An open question is whether we want to continue supporting loading demo data only for some modules?

according to my knowledge there is only place we use force_demo
<img width="414" alt="Screenshot 2024-11-29 at 13 30 59" src="https://github.com/user-attachments/assets/f334c808-bf0c-4457-9962-7e78af92a0a2">
which will reinstall all demo data

TBD: I think it would be simpler to ask users to call `force_demo` or `load_demo` function directly instead of call `load_module(..., force_demo=True)` which currently has a hard-to-define behavior